pkg/client/unversioned/jobs.go

```diff
@@ -85,7 +89,7 @@ func (c *jobs) Delete(name string, options *api.DeleteOptions) (err error) {
 		return c.r.Delete().Namespace(c.ns).Resource("jobs").Name(name).Do().Error()
 	}
 
-	body, err := api.Scheme.EncodeToVersion(options, c.r.APIVersion())
+	body, err := api.Scheme.EncodeToVersion(options, latest.GroupOrDie("").GroupVersion)
```

Comment:
$ ./kubectl delete jobs/pi
W0928 12:12:13.210531   71463 helper.go:232] Server does not support API version 'experimental/v1alpha1'. Falling back to 'v1'.